shootout/shootout.py

```diff
@@ -0,0 +1,385 @@
+        for index, session in enumerate(sessions):
+            if session["Channel"] == the_session["Channel"]:
+                sessions[index] = the_session
+        await self.config.guild(ctx.guild).Sessions.set(sessions)
+
+    async def clear_session(self, ctx) -> None:
+        sessions = await self.config.guild(ctx.guild).Sessions()
+        for index, session in enumerate(sessions):
+            if session["Channel"] == ctx.channel.id:
+                sessions.remove(session)
+        await self.config.guild(ctx.guild).Sessions.set(sessions)
+
+    @commands.command()
+    @commands.guild_only()
+    async def shootout(self, ctx):
+        """Start or join a shootout!
+
+        A shootout is a game where players can join by putting up money into the pot (if configured).
+
+        The bot will send a message to chat saying to begin and what message to type to win.
+
+        First person who types the message wins the entire pot! (Even if that pot is 0).
+
+        The game will not start if no other players have joined, and any cost will be refunded.
+
+        There is no maximum number of players.
+        """
+        settings = await self.config.guild(ctx.guild).all()
+        if await self.game_checks(ctx, settings):
+            await self.add_player(ctx, settings["cost"])
+
+    @commands.group()
+    @commands.guild_only()
+    @checks.mod_or_permissions(manage_guild=True)
+    async def soset(self, ctx):
+        """Change settings to Shootout"""
+        pass
+
+    @soset.command(name="bet")
+    async def soset_bet(self, ctx, amount: int):
+        """Sets the bet amount for a shootout"""
+        if amount < 0:
+            return await ctx.send("Get your negativity out of here!")
+        await self.config.guild(ctx.guild).cost.set(amount)
+        await ctx.tick()
+        return
+
+    @soset.command(name="draw")
+    async def soset_draw(self, ctx, *, draws: str = None):
+        """Sets the message that the bot listens for during shootout games.
+        One will randomly be selected per game.
+        """
+        if draws is None:
+            messages = await self.config.guild(ctx.guild).Messages()
+            result = ""
+            for message in messages:
+                result = result + message + "\n"
+            return await ctx.send(result)
+        await self.config.guild(ctx.guild).Messages.set(shlex.split(draws))
+        await ctx.tick()
+        return
+
+    @soset.group(name="victory")
+    async def soset_victory(self, ctx):
+        """Root command for the random victory messages."""
+        pass
+
+    @soset_victory.command(name="list")
+    async def soset_victory_list(self, ctx):
+        """List all the victories."""
+        victories = await self.config.guild(ctx.guild).Victory()
+
+        msg = "Victories:\n\n"
+        partial = []
+        for index, victory in enumerate(victories, start=1):
+            partial.append("{}. {}".format(index, victory))
```

Comment:
You should put everything in a box using the box() chat formatting. You should also pagify in case discord message limit is reached.

Reply:
Changed to 

https://github.com/shroomdog27/brandons-cogs/blob/fb5d2d6900497777a79ac3cd7e04741a12d9630e/shootout/shootout.py#L266-L268